### ISSUE TITLE
dark theme should be part of customisation, not accessibility

### ISSUE THREAD

**Issue by krodelabestiole** [nextcloud/server#17813]
hello, I've just tweaked my theme using _theming_ and _custom css_ apps.

but since I created a dark theme, I would like to apply the dark theme option I found in _accessibility_ to everyone using the Nextcloud instance, not just to my account.

I think it is closely related to the theme itself (light icons / light text so it can be seen on a dark background) and I see no reason for it not to be a customisation option that I can "force" to everyone like the rest of the theme.

for anyone interested, here is my custom CSS :
```
:root {
    --color-primary: #ff4e00;
    --color-primary-text: #000;
    --color-main-background: #3a3736;
    --color-main-background-translucent: rgba(58, 55, 54, 0.9);
}
header#header {
    --color-primary-text: #2e2e2e;
    background-color: #2e2e2e!important;
}
header#header span {
    --color-primary: #ccc;
}
```
and the main color is #5F5856
(globally it is dark gray / brownish)

**Comment by testman42** [comment]
absolutely this :point_up: 
I just found out about the location of this setting, and it gave me some quite unpleasant feelings
also it does not help that search results for "nextcloud dark theme" in any of the popular search engines show Breeze Dark and other themes, but nothing about this option

**Comment by skjnldsv** [comment]
Fixed on 25
